andela/ah-92explorers-frontend
# Users Should Be Able To Read And Delete An Article

#### What does this PR do?
- adds read and delete an article functionalities

#### Description of Task to be completed
- enables both authenticated and non-authenticated user to read article
- enables article author to delete and update article

#### How should this be manually tested?
```bash
git clone https://github.com/andela/ah-92explorers-frontend.git
git checkout ft-read-delete-article-167971932
npm install
npm run dev
```
- Before `npm run dev` add to your .env file:
```
APP_URL_BACKEND=
CRYPTO_PASSWORD=20562d21740418a343cb7e4f619ef914(should not change)
CLOUDINARY_URL=
DEFAULT_IMAGE=
```
- Navigate to `localhost:<port>/article/:<article-slug>`

#### Any background context you want to provide?
- None

#### What are the relevant pivotal tracker stories?
[#167971932](https://www.pivotaltracker.com/story/show/167971932)

#### Screenshots (if appropriate)
![Imgur](https://i.imgur.com/ma9SGQT.png)

#### Checklist
